USERS.md

```diff
@@ -1,4 +1,5 @@
 # Meet the users of Terminal theme!
+- https://blog.jontes.page **Jonte** (general tech geek)
```

Comment:
Shouldn't you add this to the bottom of the list?

Reply:
Oh, I'm sorry, I thought it was sorted from newest added to oldest. I'll fix.

Reply:
I think I fixed that now... not sure if I did it correctly. Mind reviewing again?

Reply:
It says there are conflicts.

Reply:
@Jontes-Tech I would assume that since you originally raised the PR, there have been other people to add themselves into the list.

You may want to merge in master into your branch and ensure your link is still there.

Reply:
Oh, yeah.